sycl/include/CL/sycl/detail/accessor_impl.hpp

```diff
@@ -13,6 +13,7 @@
 #include <CL/sycl/detail/sycl_mem_obj_i.hpp>
 #include <CL/sycl/id.hpp>
 #include <CL/sycl/range.hpp>
+#include <CL/sycl/stl.hpp>
 
 #include <memory>
```

Comment:
Do we still need this include?

Reply:
@bader we still use `std::make_shared`. There's no SYCL alternative now, AFAIK.

Reply:
I'm a bit lost. What is the purposed of this patch? Could you elaborate more on that in the description? 

Do you really need to use `std::make_shared`? I thought this utility simplifies the pointer constructions, but you call it directly. Right?